src/main/java/seedu/address/logic/commands/schedule/ClearScheduleCommand.java

```diff
@@ -23,7 +23,7 @@ public CommandResult execute(Model model) {
         requireNonNull(model);
         List<Candidate> candidatesToSetComplete = model.getExpiredInterviewCandidates();
         for (Candidate c : candidatesToSetComplete) {
-            model.setCandidate(c, c.setCompleted());
+            model.setCandidate(c, c.triggerInterviewStatusCompleted());
```

Comment:
Thanks for making the change! Did this late last night so I didn't see it hehe 😅

Reply:
No worries !! Thanks for doing this feature up its good!!